babel.config.js

```diff
@@ -0,0 +1,47 @@
+/* global module */
+
+module.exports = function (api) {
+  const node = {
+    ignore: [
+      'src/lib/modules/pipeline/babel-loader-overrides.js',
+      'src/lib/modules/pipeline/webpack.config.js'
+    ],
+    plugins: [
+      'babel-plugin-macros',
+      [
+        '@babel/plugin-proposal-decorators', {
+          legacy: true
+        }
+      ],
+      '@babel/plugin-syntax-dynamic-import',
+      'babel-plugin-dynamic-import-node',
+      [
+        '@babel/plugin-proposal-class-properties', {
+          loose: true
+        }
+      ],
+      [
+        '@babel/plugin-transform-runtime', {
+          corejs: 2
+        }
+      ]
+    ],
+    presets: [
+      [
+        '@babel/preset-env', {
+          targets: {
+            node: '8.11.3'
```

Comment:
Aren't we targeting 10.x ?

Reply:
From package.json:
```
"engines": {
    "node": ">=8.11.3",
    "npm": ">=6.4.1"
  }
```

Reply:
At some point I think we can target even older node versions (6.x, maybe 4.x) since those versions are still in wide use among developers, but it's not realistic to attempt that until we have much higher code coverage of embark internals and more full-featured integration testing of DApps.